src/Relation/Binary/Core.agda

```diff
@@ -192,6 +199,32 @@ P Respectsˡ _∼_ = ∀ {y} → (flip P y) Respects _∼_
 _Respects₂_ : Rel A ℓ₁ → Rel A ℓ₂ → Set _
 P Respects₂ _∼_ = (P Respectsʳ _∼_) × (P Respectsˡ _∼_)
 
+
+_Respects2_ : Rel A ℓ₁ → Rel A ℓ₂ → Set _
+P Respects2 _~_ = ∀ {x x' y y'} → x ~ x' → y ~ y' → P x y → P x' y'
```

Comment:
So I agree with you that this type is easier to use than the existing `_Respects₂_`. However a pull request about `Bin` is not the place to introduce this type. Furthermore there's still lots of existing questions about what to do with the old type and what it's new name should be etc. see #31.

Feel free to open a new PR with this in at a later point, but please can you remove it from this PR.

Reply:
Removed.